internal/cmd/beta/beta_cmd.go

```diff
@@ -54,6 +70,8 @@ All commands use the runme.yaml configuration file.`,
 	pFlags := cmd.PersistentFlags()
 	pFlags.StringVar(&cFlags.filename, "filename", "", "Name of the Markdown file to run blocks from.")
 	pFlags.StringSliceVar(&cFlags.categories, "category", nil, "Run blocks only from listed categories.")
+	pFlags.BoolVar(&cFlags.silent, "silent", false, "Silent mode. Do not error messages.")
```

Comment:
```suggestion
	pFlags.BoolVar(&cFlags.silent, "silent", false, "Silent mode. Do not print error messages.")
```

Reply:
Ha! I'm pretty sure I stuck with the copilot suggestion :-/.